Client/Frontend/Settings/ClearPrivateDataTableViewController.swift

```diff
@@ -49,7 +49,7 @@ class ClearPrivateDataTableViewController: UITableViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        title = NSLocalizedString("Clear Private Data", tableName: "ClearPrivateData", comment: "Navigation title in settings.")
+        title = NSLocalizedString("settings.clear.private.data.title", value: "Clear Private Data", tableName: "ClearPrivateData", comment: "Navigation title in settings.")
```

Comment:
At this point it would be even safer to have 3 separate strings: one for the menu item in settings, one for the title in the dialog displayed selecting this item, one for the button label.

One reason to separate them is that the available space is different between the two, and headers use a larger font, so you could use a shorter text for the title if needed.